app/src/lib/ui/ProfessionalList/List.svelte

```diff
@@ -23,7 +23,22 @@
 	const deleteAccount = mutation(deleteAccountMutation);
 
 	async function removeProfessional(professional: Professional) {
-		await deleteAccount({ accountId: professional.account.id });
+		const dateObj = new Date();
+		const month = dateObj.getUTCMonth() + 1;
+		const day = dateObj.getUTCDate();
+		const year = dateObj.getUTCFullYear();
+		const hours = dateObj.getHours();
+		const minutes = dateObj.getMinutes();
```

Comment:
Ça ne changera sûrement pas grand-chose mais c'est étonnant à la lecture de ne pas avoir `UTC` partout…

```suggestion
		const hours = dateObj.getUTCHours();
		const minutes = dateObj.getUTCMinutes();
```